composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "drupal/gin": "^3.0",
-        "drupal/core": "^8",
-        "os2forms/os2forms": "^2.6",
+        "drupal/core": "^8 || ^9",
+        "os2forms/os2forms": "^3.0",
         "os2forms/os2forms_forloeb": "^1.7",
```

Comment:
I think we should upgrade `os2forms/os2forms_forloeb` to `^1.8`, i.e. the latest version.